purchase_warn_message/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module add a warning popup on purchase orders to ensure warning is
+populated no only when partner is changed.
```

Comment:
```suggestion
populated not only when partner is changed.
```

Reply:
In addition, can we add more description to ease understanding? Myself, never use the warning, and don't understand until I look into the code. May be something like this?

```
When "Warnings" is enable for purchase order, each partner can be setup for the warning message.
This module add a persistent warning popup on purchase orders to ensure warning is populated not only when partner is changed.
```